### ISSUE TITLE
paddle.distribution.Categorical文档中对输入logits的描述不准确

### ISSUE THREAD

**Issue by loulouzny** [PaddlePaddle/Paddle#54731]
### 文档链接&描述 Document Links & Description

paddle.distribution.Categorical的行为和pytorch不一样，但文档中的对应描述没有体现出来，容易对用户产生误导。

paddle.distribution.Categorical中的输入变量名称为logits，按照pytorch与ML的一般理解，logits与prob的关系为prob=softmax(logits)。但paddle代码里面实际操作为self._prob = self.logits / dist_sum，所以这里其实不是logits，而是对应到torch的Categorical的probs。

同时，paddle.distribution.Categorical在采样的时候又是根据softmax后的结果进行的采样，所以感觉这里面有bug。不知道我的理解是否正确。

**Comment by GuoxiaWang** [comment]
问题重复 https://github.com/PaddlePaddle/Paddle/issues/54733
